firefox-ios/Ecosia/Core/Environment/URLProvider.swift

```diff
@@ -8,10 +8,11 @@ public enum URLProvider {
 
     case production
     case staging
+    case debug
 
     public var root: URL {
         switch self {
-        case .production:
+        case .production, .debug:
```

Comment:
Why is debug together with production in most cases instead of staging?

Reply:
`production` and `debug` have been running under the same `Ecosia` scheme.
The `EcosiaBeta`  scheme is being used as the only one pointing to `staging` by default at the moment.
Therefore, I thought would be easier to default to the `production` URLs for `debug` too.